### PR TITLE
Update about.html

### PR DESCRIPTION
**Pull request by PatCapon39**
Corrections to acknowledgement statement and operational partners

### CODE REVIEW

**Comment by neoformit** [comment]
Hey @PatCapon39 is this supposed to be a draft PR?

**Comment by PatCapon39** [comment]
Yep, working on some other tweaks to push through at the same time. Do you need me to close it until it's ready?

**Comment by neoformit** [comment]
Nope, just checking!

**Review by GarethPrice-Aus — APPROVED** [review]

**Comment by PatCapon39** [comment]
@neoformit this is ok to merge in now

File: webapp/home/templates/home/snippets/about/about.html [hunk 1]
```diff
@@ -42,8 +41,10 @@ <h3 id="operational-partners">
   The service is underpinned by computational resources provided by
   <a href="https://www.unimelb.edu.au/" target="_blank">The University of Melbourne</a>,
   <a href="https://www.qcif.edu.au/" target="_blank">QCIF</a>,
-  <a href="https://pawsey.org.au/" target="_blank">Pawsey Supercomputing Centre</a>,
+  <a href="https://pawsey.org.au/" target="_blank">Pawsey Supercomputing Research Centre</a>,
+  <a href="https://nci.org.au/" target="_blank">National Computational Infrastructure</a>
   <a href="https://www.aarnet.edu.au/" target="_blank">AARNet</a>
```
Comment (neoformit) [inline webapp/home/templates/home/snippets/about/about.html:46]: These two lines need a comma at the end (see line 43)

**Review by neoformit — CHANGES_REQUESTED** [review]

File: webapp/home/templates/home/snippets/about/about.html (hunk 1 above)
Comment (PatCapon39) [reply webapp/home/templates/home/snippets/about/about.html]: Added, thanks!

**Comment by PatCapon39** [comment]
@neoformit should this have gone live on dev? I can still see the old statement at https://dev.gvl.org.au/. No stress if it wasn't meant to be up there yet.

**Comment by neoformit** [comment]
That's odd - this is what I see on dev.gvl.org.au:

image

**Comment by neoformit** [comment]
image

**Comment by PatCapon39** [comment]
Ah, my apologies - I think I hadn't refreshed it properly. All good (though I am about to fix a rogue bracket that I've spotted, will pop it into my open PR

**Pull request merged**